installer/manifests/keptn/charts/control-plane/templates/api-gateway-nginx.yaml

```diff
@@ -384,9 +385,11 @@ spec:
         rollme: {{ randAlphaNum 5 | quote }}
     spec:
       {{- include "control-plane.apiGatewayNginx.pod-security-context" . | nindent 6 }}
+      terminationGracePeriodSeconds: {{ .Values.apiGatewayNginx.gracePeriod | default 120 }}
       containers:
         - name: api-gateway-nginx
           image: {{ .Values.apiGatewayNginx.image.repository }}:{{ .Values.apiGatewayNginx.image.tag }}
+          {{- include "control-plane.dist.prestop" . | nindent 10 }}
```

Comment:
I think `control-plane.dist.prestop` was meant for the distributor (`dist` as in distributor). maybe we should either split that between distributor and services or rename it to avoid confusion.
This also goes for all other services where you made use of this helper function.